services/movie.py

```diff
@@ -0,0 +1,39 @@
+from db.models import Movie, Genre, Actor
+from django.db.models import QuerySet
+
+
+def get_movies(
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None
+) -> QuerySet:
+    queryset = Movie.objects.all()
+
+    if genres_ids:
+        queryset = queryset.filter(genres__movie__in=genres_ids).distinct()
+    if actors_ids:
+        queryset = queryset.filter(actors__movie__in=actors_ids).distinct()
```

Comment:
The filtering logic for actors is incorrect. Instead of `actors__movie__in=actors_ids`, it should be `actors__id__in=actors_ids` to filter by actor IDs.